import_ldraw.py

```diff
@@ -298,18 +306,18 @@ def parse(self, filename):
                     if tmpdate != "":
                         tmpdate = tmpdate.split()
 
-                        # TODO What is this condition for?
-                        # le717 unable to find a case where it is hit.
-                        if (tmpdate[0] == "0" and
-                            len(tmpdate) >= 3 and
-                            tmpdate[1].lower() == "!ldraw_org" and
-                            "part" in tmpdate[2].lower() and
-                            self.part_count > 1
-                        ):
-                            self.subparts.append(
-                                [filename, self.mat, self.colour]
-                            )
-                            break
+                        # LDraw brick comments
```

Comment:
Ah, this is a dirty tweak that I needed to get the most general bricks to come out nicely. Case-insensitivity actually breaks the linking (not really, it just considers certain faces of bricks as separate entities, but that is really messy...). I noticed while testing that in general "!LDRAW_ORG" is in uppercase, but when it is in lower-case it is often a sub-brick, for which I don't actually need a separate orientation. Yes, this sounds like statistics... ;-)

It's just one of those things that work better one way, without really having a clear reason. It did cause me some hours of testing, before I reached this conclusion.

Actually, finding a better way to determine if a brick is a top-level brick or not, is listed highest on my to do-list. I can imagine that, although this works fine for common bricks, there are still many cases that get messy. For the Blender-project I am working on (Eifel-tower), this at least worked great!